scripts/feature_tests/README.md

```diff
@@ -78,6 +78,13 @@ export CONTAINER_RUNTIME=docker
 export NUM_NODES=4
 ```
 
+Environment variables needed for tests running clusterctl:
+
+```bash
+export CAPM3RELEASE=v0.3.0
+export CAPIRELEASE=v0.3.4
```

Comment:
Where those vars are declared?

Reply:
My understanding is that CAPM3RELEASE and CAPIRELEASE shall be declared in in the jenkins job where this test case is run.

Reply:
For both local and CI runs, they need to be declared right before running ```make```

Reply:
I think you can export them here https://github.com/metal3-io/project-infra/blob/262156d0d1e2e74187fd3d76599158510c0dc2c9/jenkins/scripts/integration_test.sh#L45

Reply:
done in https://github.com/metal3-io/project-infra/pull/131